collectors/0/goproc.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import traceback
+
+import requests
+import sys
+import time
+
+from collectors.etc import goprocconf
+from collectors.lib import utils
+
+COLLECTION_INTERVAL_SECONDS = 15
+SKIP_MEM_KEYS = ['PauseNs', 'PauseEnd', 'BySize']
```

Comment:
Can we move these to the yml?

Reply:
The collector decides what metrics it will emit, this should not be user configurable.